### PR TITLE
1.24 release

### PR DESCRIPTION
**Pull request by tbrisker**
Should only be merged a few minutes prior to announcing 1.24 GA.

### CODE REVIEW

File: _includes/manuals/1.24/1.2_release_notes.md [hunk 1]
```diff
@@ -34,7 +34,10 @@ systemctl enable --now dynflow-sidekiq@{orchestrator,worker}
 
 #### Cockpit single sign-on
 
-Cockpit single sign-on via "Web Console" button. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
+Cockpit single sign-on is now available via the "Web Console" button on the host view page when Remote Execution is installed and configured for it. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
+
+#### New status API endpoints
+Two new API endpoints have been introduced, `api/ping` and `api/status`. The former provides a way of ensuring Foreman is up and running and requires no authentication. The latter requires authentication and provides additional details on the versions and statuses of various services connected to Foreman. These should be useful for users wanting to monitor the status of their Foreman server. ([#19731](https://projects.theforeman.org/issues/19731))
```
Comment (tbrisker) [inline _includes/manuals/1.24/1.2_release_notes.md]: @ofedoren does this properly describe the api endpoints you added?

Comment (ofedoren) [reply _includes/manuals/1.24/1.2_release_notes.md]: @tbrisker, the second endpoint should be `api/statuses` (in plural form).

Otherwise, yes, fully :)

Comment (tbrisker) [reply _includes/manuals/1.24/1.2_release_notes.md]: Thanks, fixed

Comment (ekohl) [reply _includes/manuals/1.24/1.2_release_notes.md]: Note we already had `/status` (not in `/api` which gives quite minimal JSON response about the system. I think that only checks the database though.

File: _includes/version.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <div class="version-marker">
   <p>
     Latest versions:
-      <a href="{{ site.baseurl }}manuals/1.23/index.html#Releasenotesfor1.23">1.23.1</a>
+      <a href="{{ site.baseurl }}manuals/1.24/index.html#Releasenotesfor1.24">1.24.0</a>
       /
-      <a href="{{ site.baseurl }}manuals/1.22/index.html#Releasenotesfor1.22">1.22.2</a>
+      <a href="{{ site.baseurl }}manuals/1.23/index.html#Releasenotesfor1.23">1.23.1</a>
   </p>
```
Comment (lzap) [inline _includes/version.html:7]: I think you can use `/latest`/ link: https://theforeman.org/manuals/latest/index.html

It currently redirect to 1.23 tho, we should fix this as we use this in the main menu.

**Pull request merged**